docs/release_notes.rst

```diff
@@ -1,6 +1,24 @@
 Release Notes
 =============
 
+v3.4.0
+------
+
+:date: 2019-06-13
+
+This is a backwards compatible, minor release.
+
+Changes in this release:
+
+ * Adds a TTLAttribute that dynamo uses to delete records
+ * Enables TTL deletion on a dynamo table if the corresponding model has a TTLAttribute
+ * default_for_new sets default values only for unsaved objects
```

Comment:
```suggestion
 * Adds a default_for_new parameter to Attribute which is a default that applies to new items only
```

Reply:
Is this too much detail?

Reply:
I agree, let's ~remove~ the part in parenthesis